examples/complete/locals.tf

```diff
@@ -5,16 +5,12 @@ locals {
   name             = "example-complete-secret"
   filename         = "mysql-lambda.zip"
   cidr_block       = "192.168.0.0/16"
```

Comment:
Move these to the variables as well, so that the locals block will be used for dynamic data transformation e.g combining data from data source and variables.